src/encoded/static/components/genome_browser.js

```diff
@@ -3,6 +3,7 @@ import PropTypes from 'prop-types';
 import _ from 'underscore';
 import { BrowserFeat } from './browserfeat';
 import { filterForVisualizableFiles } from './objectutils';
+import Tooltip from '../libs/ui/tooltip';
```

Comment:
Is it safe to assume mobile in not a concern? The swash is distorted on mobile.

Reply:
This should be fixed, thank you for pointing it out!